bin/diagnostic.js

```diff
@@ -1 +1,12 @@
 'use strict';
+//We'll learn about require later in the course
+const ask = require('../lib/ask.js');
+
+let answer = 1;
+  answer = ask("Guess my number.");
+while (answer !== 1) {
+
+  if (answer<1 && answer>10){
```

Comment:
The condition ```answer < 1 && answer > 10``` will not work since it's impossible for ```answer``` to simultaneously be less than 1 and greater than 10.